src/Api/Persistent.hs

```diff
@@ -1,11 +1,11 @@
-{-# LANGUAGE TypeOperators               #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving  #-}
 {-# LANGUAGE ImpredicativeTypes          #-}
 {-# LANGUAGE OverloadedStrings           #-}
 {-# LANGUAGE ScopedTypeVariables         #-}
 {-# LANGUAGE TemplateHaskell             #-}
+{-# LANGUAGE TypeOperators               #-}
 
-{-# OPTIONS_GHC -Wall -Werror #-}
+{-# OPTIONS_GHC -Werror #-}
```

Comment:
You could also add `-fno-warn-orphans` and leave `-Wall` intact, but it's all good.  We have travis.